docs/sdks/client-sdks/ios/assignments.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Assignments
+sidebar_position: 4
+---
+
+import ApiOptionRef from '@site/src/components/ApiOptionRef';
+
+Assignments are the mechanism through which a given [Subject](/sdks/sdk-features/subjects) is assigned to a variation for a feature flag or experiment.
+
+## How Assignments Work
+
+The SDK periodically retrieves configuration rules from the Eppo server that define how subjects should be allocated to variants. When you call an assignment function, the SDK evaluates these rules locally without making additional network requests.
```

Comment:
No polling in Swift - configuration loaded once on session start and whenever user calls `load()`